### ISSUE TITLE
More sharing options for topics

### ISSUE THREAD

**Issue by loorm** [citizenos/citizenos-fe-old#313]
**What is the problem?**
Śharing on social media is the norm. When you open up share dialog in topic view, you expect to be able to share to different social media platforms. Agreed, you can do so via the link you are given, but it would improve usability to have a selection of share buttons there.

**Why is this a problem?**
Involving people into topic discussions via social media sharing is vital to the functioning of COS concept. Without easily usable share options, it's difficult to bring people into discussions.

**Possible solution.**
See image comparison below.

Our Share dialog:
![social_share](https://user-images.githubusercontent.com/22661743/67857601-e40b1400-fb1f-11e9-82f0-9781fe88a646.JPG)

ERR Share dialog:
![good_social_share](https://user-images.githubusercontent.com/22661743/67857606-e9685e80-fb1f-11e9-8170-b702a931f675.JPG)

**Comment by loorm** [comment]
Triage 14 - this is linked to #306, which needs to be done first. @kevincrepin - can you please do the visual for this?

**Comment by kevincrepin** [comment]
@loorm we can add social sharing to the share options. But I would also say that right now the sharing is too hidden. The user already has to go find it under the settings or in the options menu. It's not standing out very much, so should it (now or in the future?)

Which platform can it be shared to btw, currently I put Facebook, Linkedin and Twitter.

![sharing](https://user-images.githubusercontent.com/26792281/68124172-611afc80-ff17-11e9-8ed1-b5b0e63cf0d4.png)

**Comment by loorm** [comment]
Duplicate of #512. Closing.
